app/config/config.yml

```diff
@@ -376,6 +376,21 @@ nelmio_cors:
             max_age: 3600
 
 api_platform:
-    mapping:
-        paths: ['%kernel.project_dir%/src/Entity']
-    enable_docs: '%kernel.debug%'
+    name_converter: serializer.name_converter.camel_case_to_snake_case
+    resource_class_directories:
+        - '%kernel.project_dir%/src/Entity'
+    enable_docs: false
+    enable_swagger_ui: false
```

Comment:
I think it is not very useful as you already disable the doc with `enable_docs: false`, WDYT?

Reply:
Done.